lib/Yam/Agama/Pom/GrubMenuAgamaPage.pm

```diff
@@ -15,14 +15,13 @@ sub new {
     my ($class, $args) = @_;
     return bless {
         grub_menu_base => $args->{grub_menu_base},
-        tag_agama_installer_highlighted => 'grub-menu-agama-installer-highlighted',
+        tag_install_product_entry_highlighted => 'grub-menu-agama-installer-highlighted',
+        tag_boot_from_hard_disk => 'inst-bootmenu-boot-harddisk',
     }, $class;
 }
 
 sub expect_is_shown {
-    my ($self) = @_;
-    send_key_until_needlematch($self->{tag_agama_installer_highlighted}, 'down') unless check_screen($self->{tag_agama_installer_highlighted}, 10);
-    assert_screen($self->{tag_agama_installer_highlighted}, 60);
+    record_info('New', 'New entry but boot from harddisk as default');
```

Comment:
this needs to be reverted to its original code as well, to check for the first entry needle, regardless of which one it is. The goal of this method is just to know that you are in the screen, so the logo could even be ok, in principle we could just put under the same tag two different needles for the two cases.